src/main/java/org/zalando/nakadi/plugin/api/ApplicationService.java

```diff
@@ -1,5 +1,7 @@
 package org.zalando.nakadi.plugin.api;
 
+import org.zalando.nakadi.plugin.api.exceptions.PluginException;
```

Comment:
These interfaces import `PluginException` but no method mentions it explicitly. Isn't the idea to add it to the method signature?

Reply:
added to Signature